index.html

```diff
@@ -411,385 +410,88 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
-<aside class="note">It is possible for <a data-link-for=
-"PerformanceResourceTiming">transferSize</a> value to be lower than
-<a data-link-for="PerformanceResourceTiming">encodedBodySize</a>:
-when a cached response is successfully revalidated the
-<a data-link-for="PerformanceResourceTiming">transferSize</a>
-reports the size of the response HTTP headers incurred during the
-revalidation, and <a data-link-for=
-"PerformanceResourceTiming">encodedBodySize</a> reports the size of
-the previously retrieved payload body.</aside>
-</li>
-<li>zero otherwise, including for resources retrieved from
-<a data-cite="HTML#relevant-application-cache">relevant application
-caches</a> or from local resources.</li>
-</ol>
-<p data-dfn-for="PerformanceResourceTiming">On getting, the
-<dfn>encodedBodySize</dfn> attribute MUST return as follows:</p>
-<ol>
-<li>The size, in octets, received from a <a data-cite=
-"FETCH#http-network-or-cache-fetch">HTTP-network-or-cache</a>
-fetch, of the <a data-cite="RFC7230#section-3.3">payload body</a>
-[[RFC7230]], prior to removing any applied <a data-cite=
-"RFC7231#section-3.1.2.1">content-codings</a> [[RFC7231]], if the
-resource passes the <a>timing allow check</a> algorithm.</li>
-<li>The size, in octets, of the <a data-cite=
-"RFC7230#section-3.3">payload body</a> prior to removing any
-applied <a data-cite="RFC7231#section-3.1.2.1">content-codings</a>
-if the resource is retrieved from <a data-cite=
-"HTML#relevant-application-cache">relevant application caches</a>
-or from local resources.</li>
-<li>zero, otherwise.</li>
-</ol>
-<aside class="note">The <code>encodedBodySize</code> may be zero
-depending on the <a data-cite="RFC7231#section-6">response code</a>
-- e.g. <a data-cite="RFC7231#section-6.3.5">HTTP 204 (No
-Content)</a>, <a data-cite="RFC7231#section-6.4">3XX</a>,
-etc.</aside>
-<p data-dfn-for="PerformanceResourceTiming">On getting, the
-<dfn>decodedBodySize</dfn> attribute MUST return as follows:</p>
-<ol>
-<li>The size, in octets, received from a <a data-cite=
-"FETCH#http-network-or-cache-fetch">HTTP-network-or-cache</a>
-fetch, of the <a data-cite="RFC7230#section-3.3">message body</a>
-[[RFC7230]], after removing any applied <a data-cite=
-"RFC7231#section-3.1.2.1">content-codings</a> [[RFC7231]], if the
-resource passes the <a>timing allow check</a> algorithm.</li>
-<li>The size, in octets, of the <a data-cite=
-"RFC7230#section-3.3">payload</a> after removing any applied
-<a data-cite="RFC7231#section-3.1.2.1">content-codings</a>, if the
-resource is retrieved from <a data-cite=
-"HTML#relevant-application-cache">relevant application caches</a>
-or from local resources.</li>
-<li>zero, otherwise.</li>
+<p data-dfn-for="PerformanceResourceTiming">When <dfn>toJSON</dfn>
+is called, run [[WEBIDL]]'s <a data-cite=
+"WEBIDL#default-tojson-operation">default toJSON operation</a>.</p>
+<p data-dfn-for="PerformanceResourceTiming">
+<dfn>initiatorType</dfn> getter steps would be to return the
```

Comment:
nit: "The initiatorType getter steps are to return", for consistency with the rest?

Reply:
fixed